lib/navigation.php

```diff
@@ -418,7 +418,8 @@ function gutenberg_register_navigation_post_type() {
 		'description'           => __( 'Navigation menus.', 'gutenberg' ),
 		'public'                => false,
 		'has_archive'           => false,
-		'show_ui'               => true,
```

Comment:
I’m not sure whether we should disable `wp_navigation`  definition entirely or just the `show_ui`  flag. My PR only disables the `show_ui`  flag.

Reply:
When in doubt, do the safe thing. `show_ui` it is for now.